projects/storefrontlib/src/lib/store-finder/components/store-finder-grid/store-finder-grid.component.spec.ts

```diff
@@ -4,21 +4,19 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { StoreModule } from '@ngrx/store';
```

Comment:
We will remove 'StoreModule' from all component.spec. You can see how other component.specs are implemented, e.g. `mult-step-checkout.component.spec.ts`.

Reply:
I removed all StoreModule-s imports in unit tests.